src/System.Composition.Hosting/src/ILLinkTrim.xml

```diff
@@ -0,0 +1,22 @@
+<linker>
+  <assembly fullname="System.Composition.Hosting">
+    <type fullname="System.Composition.Hosting.Providers.ExportFactory.ExportFactoryExportDescriptorProvider">
```

Comment:
Also, I do plan to add comments for every case the linker misses along with a reason why it can't have a heuristic.